packages/api-client/src/main/event/UserEvent.ts

```diff
@@ -54,6 +55,11 @@ export interface UserClientAddEvent extends UserEvent {
   type: USER_EVENT.CLIENT_ADD;
 }
 
+export interface UserClientAddNotification {
```

Comment:
It seems the naming is inconsistent now. Some interfaces are Events and some are Notifications now. Is there a difference or is it the same?

Reply:
It's on purpose. I put the notifications next to the events because they are similar and it's an indication for us to check if every new event also has a matching notification in the notification stream.

Reply:
We discussed this in 1:1 and came to the conclusion to rename the `*Events` to something else because these are actually no events. Instead they are synchronous responses to REST calls.

Reply:
https://github.com/wireapp/wire-pwa/issues/1321